settings.gradle

```diff
@@ -26,3 +26,7 @@ include 'TabHostExample'
 include 'TabHostSelfContainedTabBrowsing'
 include 'wizard'
 include 'EditTextChips'
+include 'WearBuildConfigMobile'
+project(':WearBuildConfigMobile').projectDir = new File('WearBuildConfig/mobile')
```

Comment:
What I'm trying to do is to create two modules that share the same parent in the main project root:

- WearBuildConfig/mobile
- WearBuildConfig/wear

Can you specify what's wrong with this way of doing it?
This is the same strategy it was used for other modules here and I'll be curious to hear about alternatives (even if we might make this project independent)